### ISSUE TITLE
Accessibility: [Two UI]: [Choice group]: In High contrast black disabled option radio button text is looks like regular text.

### ISSUE THREAD

**Issue by SowmyaNat** [microsoft/fluentui#13767]
===== Comments Moved by Partner Bug Service 06/16/2020 09:15 PM UTC =====

Platform:

OS - Windows 10-Version 1909- (Build 18363.778)

Microsoft Edge Dev Build 

Version 84.0.502.0  (Official build) dev (64-bit)

Pre-requisites: 

High contrast black -Turn on

Repro steps:

1.Load https://account.microsoft.com/internal/twoui/iframe.html?id=common-choicegroup--when-an-option-is-disabled&viewas=engineer in edge browser with valid credentials.

2. Select option B.

3.Observe Disable option in High contrast black.

Expected Result:

Disable option should be in Green color.

Actual Result:

Disable option is visible as regular option. User is unable to find disable option in choice group.

Note:

This behavior was observed in Edge Chromium browser.
![Disale option in Edge chromium browser](https://user-images.githubusercontent.com/67346235/85471407-46100200-b565-11ea-8ef0-6ee644177b69.JPG)
![Actual behaviour in Edge Spartan](https://user-images.githubusercontent.com/67346235/85471420-4ad4b600-b565-11ea-8482-a071420f1471.JPG)

**Comment by paulgildea** [comment]
@xugao adding to #11742 list.
